public/graphing-calculator.html

```
@@ -0,0 +1,1166 @@
+          <div class="flex gap-2">
+            <input
+              id="eq-input"
+              type="text"
+              placeholder="e.g., y = sin(x)"
+              class="flex-1 min-w-0 text-sm rounded-lg px-3 py-2.5 bg-gray-50 dark:bg-gray-700/80 border border-gray-200 dark:border-gray-600 text-gray-800 dark:text-gray-100 placeholder-gray-400 focus:outline-none focus:ring-2 focus:ring-teal-400 transition-colors"
+              autocomplete="off" spellcheck="false"
+              aria-label="Enter equation expression"
+              onkeydown="if(event.key==='Enter') addFromInput()"
+            />
+            <button
+              onclick="addFromInput()"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)